gradle.properties

```diff
@@ -14,4 +14,6 @@
 
 #The base URL of the server rest API. Must end with /
 dribbbleApiUrl = https://api.dribbble.com/v1/
-accessToken = UNDEFINED
\ No newline at end of file
+accessToken = UNDEFINED
+
+android_support_library_version = 23.3.0
```

Comment:
_Minor comment! feel free to ignore._

What is the reason for changing property name from `SUPPORT_LIBRARY_VERSION` to `android_support_library_version`?
For consistency with other property name, should have used the same one.